polyfills/resolveGlobal.js

```diff
@@ -0,0 +1,7 @@
+export default function resolveGlobal() {
```

Comment:
Q: The reason we don't have a test for this, is it because it is hard to test without an actual window or global? Could we mock these? 

Q: I'm also having prettier complaining about the formatting of this file, did a clean install though

Reply:
If prettier complains than it means that the file is incorrect. But I think you mean that ESlint is complaining.

Reply:
@RobinMalfait true

Reply:
I can add a test but the thing is we can set Jest to a certain environment either 'node' or 'jsdom' which is global or window. The thing is what do we check then? That the result is window or global? But we can't do that since the environment is also setup with a mock.

Manually mocking is hard to do, we can use `with()` which resets the context but that is not allowed in strict mode and all the tests in jest run in strict mode.
So what we can do is mock `global.window` and `global.global` but the problem there is that `global.window` and `global.global` and `global` will all be the same reference since this is the jest environment. Though spot.

> **NOTE:** The code written in `resolveGlobal` is not something I came up with. It is the default `commonjsGlobal` detection thingy (https://github.com/search?q=commonjsGlobal&type=Code)

Reply:
yes I know about the commonjsGlobal stuff, but was wondering why we didn't have a test for this; and was thinking that it would be hard to do. Thanks for the clarification.